components/throttle.js

```diff
@@ -0,0 +1,93 @@
+// throttle.js
+
+var _ = require('underscore');
+var fs = require('fs');
+
+var idIndex = 0;
```

Comment:
What happens when there are multiple throttle nodes in the same network (sharing the same idIndex, but different file_envvar)?